File: Exercicios/Python/Exercicio06.py
somaPares = 0
somaImpares = 0
somaDiv3 = 0

for x in range(10):
    num = int(input("Digite um numero: "))
    if num%2 == 0:
        somaPares += num
    else:
        somaImpares += num
    if num%3 == 0:
        somaDiv3 += num
print("A soma dos numeros pares é: " + str(somaPares))
print("A soma dos numeros impares é: " + str(somaImpares))
print("A soma dos numeros divisiveis por três é: " + str(somaDiv3))

File: Exercicios/Java/Exercicio06.java
import java.util.Scanner;

public class Exercicio06 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num, somaPares = 0, somaImpares = 0, somaDiv3 = 0;

        for(int i = 0; i < 10; i++){
            System.out.print("Digite um numero: ");
            num = sc.nextInt();
            if(num%2 == 0){
                somaPares += num;
            } else {
                somaImpares += num;
            }
            if(num%3 == 0){
                somaDiv3 += num;
            }
        }
        sc.close();
        System.out.println("A soma dos numeros pares é: " + somaPares);
        System.out.println("A soma dos numeros impares é: " + somaImpares);
        System.out.println("A soma dos numeros divisiveis por três é: " + somaDiv3);
    }
}
